style.css

```diff
@@ -148,6 +148,428 @@ body {
+  font-style: normal;
+  font-weight: bold;
+  font-size: 10px;
+  line-height: 10px;
+  letter-spacing: .07em;
+  color: #000;
+  opacity: .5;
+}
+
+button {
+  background: none;
+}
+
+.middle-section__btn,
+.middle-section__btn_other {
+  text-transform: uppercase;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 13px;
+}
+
+.middle-section__title {
+  font-style: normal;
+  font-weight: bold;
+  font-size: 20px;
+  line-height: 20px;
+  color: #000000;
+}
+
+.middle-section__desription {
+  font-style: normal;
+  font-weight: normal;
+  font-size: 9px;
+  line-height: 13px;
+  color: #000000;
+  opacity: 0.5;
+}
+
+.bottom-section {
+  position: relative;
+  padding: 33px 12px;
+  background: -webkit-gradient(linear, left top, right top, color-stop(27%, #ebe3d8), color-stop(27%, #000), color-stop(38%, #000), color-stop(38%, #ebe3d8));
+  background: -webkit-linear-gradient(left, #ebe3d8 27%, #000 27%, #000 38%, #ebe3d8 38%);
+  background: -o-linear-gradient(left, #ebe3d8 27%, #000 27%, #000 38%, #ebe3d8 38%);
+  background: linear-gradient(to right, #ebe3d8 27%, #000 27%, #000 38%, #ebe3d8 38%);
+}
+
+.bottom-section__image {
+  z-index: -50;
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-pack: center;
+  -ms-flex-pack: center;
+  justify-content: center;
+}
+
+.bottom-section__img {
+  width: 100%;
+  -webkit-box-shadow: -11px 25px 40px -34px rgba(0, 0, 0, .8);
+  box-shadow: -11px 25px 40px -34px rgba(0, 0, 0, .8);
+}
+
+.bottom-section__img img {
+  width: 100%;
+  height: 100%;
+  -o-object-fit: contain;
+  object-fit: contain;
+}
+
+.bottom-section__block {
+  z-index: 50;
+  position: relative;
+  top: -24px;
+  margin-left: 32px;
+  margin-right: 14px;
+  padding: 20px 25px;
+  background: #FAE8E4;
+}
+
+.bottom-section__title,
+.bottom-section__subtitle {
+  margin: 0;
+  margin-top: 5px;
+}
+
+.bottom-section__description {
+  margin-bottom: 22px;
+  opacity: .5;
+}
+
+.bottom-section__btn_other {
+  position: relative;
+  width: 138px;
+  padding-bottom: 30px;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: normal;
+  cursor: pointer;
+  text-transform: uppercase;
+  text-decoration: none;
+  color: #000;
+}
+
+.bottom-section__arrow {
+  cursor: pointer;
+  position: relative;
+}
+
+.bottom-section__arrow div {
+  position: relative;
+  width: 108px;
+  height: 2px;
+  background-color: #000;
+  display: block;
+}
+
+.bottom-section__arrow div::after {
+  content: '';
+  position: absolute;
+  width: 7px;
+  height: 1px;
+  top: -3px;
+  right: 0;
+  background-color: #000;
+  transform: rotate(45deg);
+}
+
+.bottom-section__arrow div::before {
+  content: '';
+  position: absolute;
+  width: 7px;
+  height: 1px;
+  top: 4px;
+  right: 0;
+  background-color: #000;
+  box-shadow: 0 3px 5px rgba(0, 0, 0, .2);
+  transform: rotate(-45deg);
+}
+
+.bottom-section__btn_basket {
+  height: 30px;
+  text-transform: uppercase;
+  border: 1.5px solid #000000;
```

Comment:
Codacy found an issue: [Expected "#000000" to be "#000" (color-hex-length)](https://app.codacy.com/gh/WebHeroSchool/locoalin/pullRequest?prid=5835770)